addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComThermostat3MessageTest.java

```diff
@@ -8,21 +8,50 @@
  */
 package org.openhab.binding.rfxcom.internal.messages;
 
-import static org.openhab.binding.rfxcom.internal.messages.RFXComBaseMessage.PacketType.THERMOSTAT3;
-
 import org.junit.Test;
-import org.openhab.binding.rfxcom.internal.exceptions.RFXComMessageNotImplementedException;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+import javax.xml.bind.DatatypeConverter;
+
+import static org.junit.Assert.assertEquals;
+import static org.openhab.binding.rfxcom.internal.messages.RFXComBaseMessage.PacketType.THERMOSTAT3;
+import static org.openhab.binding.rfxcom.internal.messages.RFXComThermostat3Message.SubType.MERTIK__G6R_H4TB__G6_H4T__G6R_H4T21_Z22;
 
 /**
  * Test for RFXCom-binding
  *
- * @author Martin van Wingerden
- * @since 1.9.0
+ * @author Martin van Wingerden - Initial contribution
```

Comment:
Don't we want a @since in here, just a newer version?

Reply:
I suddenly remembered that we stopped using them. @kaikreuzer?

Reply:
correct.